README.md

```diff
@@ -425,13 +427,23 @@ If you would like to check for any active virtual media you can run ```badfish``
 badfish -H mgmt-your-server.example.com -u root -p yourpass --check-virtual-media
 ```
 
+### Mount Virtual Media
+If you would like to mount an ISO from network you can run ```badfish``` with the ```--mount-virtual-media``` option which post a request for mounting the ISO virtual media (Virtual CD). Needs two arguments: the address and the path to the ISO.
+```bash
+badfish -H mgmt-your-server.example.com -u root -p yourpass --mount-virtual-media http://storage.example.com /folder/linux.iso
```

Comment:
small typo on the URL

Reply:
Changed with the latest [commit](https://github.com/redhat-performance/badfish/pull/324/commits/8f12e5b12bf23115b7d6e798205d0cb524354180).